conans/client/command.py

```diff
@@ -1650,6 +1649,59 @@ def editable(self, *args):
                 self._user_io.out.writeln("    Path: %s" % v["path"])
                 self._user_io.out.writeln("    Layout: %s" % v["layout"])
 
+    def graph(self, *args):
+        """
+        """
+        parser = argparse.ArgumentParser(description=self.remote.__doc__,
+                                         prog="conan graph",
+                                         formatter_class=SmartFormatter)
+        subparsers = parser.add_subparsers(dest='subcommand', help='sub-command help')
+        subparsers.required = True
+
+        # create the parser for the "a" command
+        merge_cmd = subparsers.add_parser('update-lock', help='merge two lockfiles')
+        merge_cmd.add_argument('old_lockfile', help='previous lockfile folder')
+        merge_cmd.add_argument('new_lockfile', help='modified lockfile folder')
+
+        build_order_cmd = subparsers.add_parser('build-order', help='Returns build-order')
+        build_order_cmd.add_argument('lockfile', help='lockfile folder')
+        build_order_cmd.add_argument("-b", "--build", action=Extender, nargs="?", help="nodes to build")
+        build_order_cmd.add_argument("--json", action=OnceArgument,
+                                     help="generate output file in json format")
+
+        clean_cmd = subparsers.add_parser('clean-modified', help='Clean modified')
+        clean_cmd.add_argument('lockfile', help='lockfile folder')
+
+        lock_cmd = subparsers.add_parser('lock', help='create a lock file')
```

Comment:
Clarify the difference between `conan info` and `conan graph lock` and if we really need them both.

Reply:
The conan info will remain as before. It will be able to use a lockfile, as the other commands, but ``conan graph lock`` will be the mechanism to generate a lock. I think it is way more clear and not keep messing with the ``info`` command.